src/lib/TranscriptEditor/MediaPlayer/defaultHotKeys.js

```diff
@@ -1,109 +1,109 @@
 function returnHotKeys(self) {
   return {
-    'esc': {
+    'alt+space': {
       priority: 1,
       handler: () => { 
         self.togglePlayMedia(); 
         
         self.props.handleAnalyticsEvents({ 
           category: 'defaultHotKeys', 
-          action: 'esc', 
+          action: 'alt+space', 
           name: 'togglePlayMedia', 
           value: 'na'
         });
 
       },
       label: 'Play Media',
     },
-    'esc': {
-      priority: 1,
-      handler: () => { 
-        self.promptSetCurrentTime();
-        
-        self.props.handleAnalyticsEvents({ 
-          category: 'defaultHotKeys', 
-          action: 'ctrl+k', 
-          name: 'promptSetCurrentTime', 
-          value: 'na'
-        });
-
-      },
-      label: 'set current time',
-    },
-    'alt+right': {
+    'shift+right': {
       priority: 1,
       handler: () => {
         self.skipForward(); 
 
         self.props.handleAnalyticsEvents({ 
           category: 'defaultHotKeys', 
-          action: 'ctrl+1', 
+          action: 'alt+right', 
```

Comment:
I disagree with this term 🙃

Reply:
ahah, yeah, happy to change the keyboard shortcuts

They will need to be revisited, coz the main problem is that a lot of other key combinations are taken, either by the mac environment or the browser one.  (without considering the windows one) - so yeah, I guess this is just a first pass.

Reply:
suggestions for alternative keyboards shortcuts welcome